### ISSUE TITLE
Options is not optional for `replicate.wait` in TypeScript

### ISSUE THREAD

**Issue by thelinuxkid** [replicate/replicate-javascript#154]
When running 

```
  prediction = await replicate.wait(prediction);
```

using TypeScript I get an error if I don't pass empty options since `options` is not really marked as optional in the type definition.

The current work around to add an empty options

```
  prediction = await replicate.wait(prediction, {});
```

**Comment by mattt** [comment]
Hi @thelinuxkid. Thanks for reporting this. I have a fix for this in #155.

**Comment by thelinuxkid** [comment]
Thanks for the quick update Matt

**Comment by mattt** [comment]
Thanks again for reporting, @thelinuxkid. This is now fixed in [v0.21.1](https://github.com/replicate/replicate-javascript/releases/tag/v0.21.1).
